### PR TITLE
Add events

### PR DESCRIPTION
**Pull request by frempongdev**
### Added logic needed to make the Calculator component work. 🖩
  - Created a new "logic" directory
  - Added calculate.js and operate.js to the logic directory.
  - Installed big.js
  - Imported necessary calculate.js into calculator.js. ⬇️ 
  - Implemented click events using the math logic from calculate.js 🖱️ 
  - Fixed Linter errors. 🧰

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,27 +1,44 @@
...
-      <button type="button" id="plus" className="operand">+</button>
-      <button type="button" id="zero" className="num">0</button>
-      <button type="button" id="dot" className="special">.</button>
-      <button type="button" id="equal-to" className="operand">=</button>
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
+
+const initialState = { total: 0, next: '', operation: '' };
+const Calculator = () => {
+  const [state, setState] = useState(initialState);
```
Comment (OLIPLICHE) [inline src/components/Calculator.js:6]: 💯 💯 💯 

Capture

**Review by OLIPLICHE — APPROVED** [review]
Hi @frempongdev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- The App is working fine👏 👏
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (frempongdev) [reply src/components/Calculator.js:6]: Thank You.

**Comment by frempongdev** [comment]
@OLIPLICHE ,
I appreciate your efforts in going through this Project. Thanks again for approving this Milestone.

**Pull request merged**